ios-guide.md

```diff
@@ -73,18 +73,9 @@ You may also use the deprecated [application:openURL:sourceApplication:annotatio
 Because only one `openURL` method can be defined, if you have multiple listeners which should be defined (for instance if you have both Google and Facebook OAuth), you must combine them into a single function like so:
 
 ```objc
-- (BOOL)application:(UIApplication *)application openURL:(NSURL *)url
-  sourceApplication:(NSString *)sourceApplication annotation:(id)annotation {
-
-  return [[FBSDKApplicationDelegate sharedInstance] application:application
-                                                        openURL:url
-                                              sourceApplication:sourceApplication
-                                                     annotation:annotation
-         ]
-         || [RNGoogleSignin application:application
-                                openURL:url
-                      sourceApplication:sourceApplication
-                             annotation:annotation
-            ];
+- (BOOL)application:(UIApplication *)application openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options
```

Comment:
there's the same method call described above, but using a slightly different signature. Can you make sure both places use the correct one? https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623112-application?language=objc thanks!

Reply:
Both works, there's one legacy and one new. This one is new.